pkg/util/async/bounded_frequency_runner_test.go

```diff
@@ -358,6 +359,7 @@ func Test_BoundedFrequencyRunnerBurst(t *testing.T) {
 
 	// Clean up.
 	stop <- struct{}{}
+	<- time.updated
```

Comment:
maybe add a comment explaining that this is draining the channel to avoid keep the goroutine hanging after bfr.stop()

Reply:
+1